rune/libcontainer/specconv/spec_linux.go

```diff
@@ -332,11 +332,29 @@ func createEnclaveConfig(spec *specs.Spec, config *configs.Config) {
 		args = strings.Join(a, " ")
 	}
 
+	ra_client_spid := filterOut(env, "RA_CLIENT_SPID")
```

Comment:
Change the variable prefix to ENCLAVE_, and rename this variable to ENCLAVE_RA_EPID_SPID.